xbmc/profiles/ProfileManager.h

```diff
@@ -148,14 +148,22 @@ class CProfileManager : protected ISettingsHandler,
     used profile will be loaded
     \return the id to the autologin profile
     */
-  int GetAutoLoginProfileId() const { return m_autoLoginProfile; }
+  int GetAutoLoginProfileId() const
+  {
+    return m_autoLoginProfile;
+    Save();
```

Comment:
This function doesn't change any value. Also `Save()` would never get call, as it is after the return statement.
I think you actually wanted to change the `ToggleLoginScreen()` function

Reply:
Yes - I was too fast here. I changed it. Thanks much.